src/packages/formitem/formitem.tsx

```
@@ -202,34 +215,40 @@
-        className={`nut-form-item ${className}`}
+        className={`${this.getClassNameWithDirection('nut-form-item')} ${className}`}
         style={style}
         align={align}
         onClick={(e) =>
           this.props.onClick && this.props.onClick(e, this.componentRef)
         }
       >
         {label ? (
-          <div className="nut-cell-title nut-form-item-label">
+          <div
+            className={`nut-cell-title ${this.getClassNameWithDirection('nut-form-item-label')}`}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)